src/App.js

```diff
@@ -1,14 +1,40 @@
 import React from "react";
-import ProPic from "./components/portfolioPic.png";
 import "./App.css";
+import NavMenu from './components/navMenu/NavMenu'
+import Home from './components/home/Home'
+import Software from './components/software/Software'
+import Music from './components/music/Music'
+import Design from './components/design/Design'
+import Footer from './components/footer/Footer'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
 
```

Comment:
This last import from react router(Dom) is a React-Library for handling the browser history for single page web applications. (If it's one page then how can you go back and forth between states. Enter react router(Dom)...

Reply:
The "App.js" is the  application and the "Router" embeds a different component based on the current path.